doc/cli_vs_yum.rst

```diff
@@ -121,6 +121,7 @@ Also see related Fedora bugzillas `982947
  ``skip_if_unavailable`` enabled by default
 ============================================
 
+In some distribution DNF is shipped with ``skip_if_unavailable=True`` in ``/etc/dnf/dnf.conf``.
```

Comment:
Should read:
> In some distribution**s** ...

However, with or without this, I'm quite confused what this paragraph is actually trying to say... Especially with the back and forth that went on about this... Maybe it should be rephrased and start with defining what the real difference to yum actually is?

Reply:
I wish we'd just get `/etc/dnf/conf.d/*.conf` files working... Then we can ship distro override snippets.